integrations/pytorch/recipes/sparse-transfer-learn.md

```diff
@@ -0,0 +1,98 @@
+    start_epoch: 0.0
+    lr_class: MultiStepLR
+    lr_kwargs:
+      milestones: *lr_step_milestones
+      gamma: 0.1
+    init_lr: *init_lr
+
+# Phase 1 Sparse Transfer Learning / Recovery
+sparse_transfer_learning_modifiers:
+  - !ConstantPruningModifier
+    start_epoch: 0.0
+    params: __ALL__
+    
+---
+
+# Transfer Learning
+
+This recipe provides a framework for performing sparse transfer learning on
+SparseML's `sparsified` ResNet-50, or MobileNet model.  This recipe was written for transfer learning
+onto the [Imagenette](https://github.com/fastai/imagenette) dataset using
+a single GPU, Adam optimizer, and batch size 32.  Adjust the hyperparameters at the top of the file accordingly
+for your dataset and training environment.
+
+## Imagenette
+[Imagenette](https://github.com/fastai/imagenette) is a subset of 10 easily classified classes from ImageNet (tench, English springer, cassette player, chain saw, church, French horn, garbage truck, gas pump, golf ball, parachute).
+To transfer learn ResNet-50 on [Imagenette](https://github.com/fastai/imagenette) with this recipe, run the following example command.
+
+```
+python integrations/pytorch/vision.py train \
+    --recipe-path integrations/pytorch/recipes/sparse-transfer-learn.md \
+    --checkpoint-path /PATH/TO/MODEL_CHECKPOINT \
+    --arch-key resnet50 \
+    --model-kwargs '{"ignore_error_tensors": ["classifier.fc.weight", "classifier.fc.bias"]}' \
+    --dataset imagenette \
+    --dataset-path /PATH/TO/IMAGENETTE  \
+    --train-batch-size 32 --test-batch-size 64 \
+    --loader-num-workers 0 \
+    --optim Adam \
+    --optim-args '{}' \
+    --model-tag resnet50-imagenette-sparse-transfer-learned
+```
+
+To transfer learn MobileNet on [Imagenette](https://github.com/fastai/imagenette) with this recipe, run the following example command.
+
+```
+python integrations/pytorch/vision.py train \
+    --recipe-path  integrations/pytorch/recipes/sparse-transfer-learn.md\
+    --checkpoint-path /PATH/TO/MODEL_CHECKPOINT \
+    --arch-key mobilenet \
+    --model-kwargs '{"ignore_error_tensors": ["classifier.fc.weight", "classifier.fc.bias"]}' \
+    --dataset imagenette \
+    --dataset-path /PATH/TO/IMAGENETTE  \
+    --train-batch-size 32 --test-batch-size 64 \
+    --loader-num-workers 0 \
+    --optim Adam \
+    --optim-args '{}' \
+    --model-tag mobilenet-imagenette-sparse-transfer-learned
+```
+## Other Datasets
+To transfer learn this sparsified model to other datasets
+you may have to adjust certain hyperparameters in this recipe and/or training script.
+Some considerations:
+* For more complex datasets, increate the number of epochs, adjusting the learning rate step accordingly
+* Adding more learning rate step milestones can lead to more jumps in accuracy
+* Increasing the number of epochs before starting quantization can give a higher baseline
+* If accuracy is not recovering in the quantization phase, the total number of epochs should be increased as well
```

Comment:
nit: remove if recipe does not include quantization

Reply:
Eye of the tiger! 🐅